.travis.yml

```diff
@@ -34,30 +33,6 @@ before_install:
 
 jobs:
   include:
-    - stage: test
```

Comment:
If you do this I believe RMG-tests won't trigger anymore which is why it isn't triggering on this PR

Reply:
I've re-enabled the trigger.